exercise-analyst-server/API/Controllers/MeasurementsDevController.cs

```diff
@@ -28,5 +27,27 @@ public async Task<IActionResult> CreateMeasurementDev([FromBody] CreateMeasureme
 
             return BadRequest(response.Errors);
         }
+
+        [HttpGet]
+        public async Task<IActionResult> GetMeasurementsDev()
+        {
+            var response = await _measurementsDevService.GetMeasurementsAsync();
+            
+            if (response.HttpStatusCode == HttpStatusCode.OK)
+                return Ok(response);
+
+            return BadRequest(response.Errors);
```

Comment:
To nie jest osiągalne ale jeżeli ważacie, że ładnie wygląda i wgl clean code to można zostawić

Reply:
Fakt, można wywalić :D